packages/init/bin/d2-init

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env node
+console.log('a placeholder for being able to run `npx @dhis2/create` to scaffold a new app')
```

Comment:
this would just be an alias to `d2 app scripts init` but allows consumers to use the more modern standard way of scaffolding apps `npx @dhis2/create my-app`